.github/workflows/build.yml

```diff
@@ -0,0 +1,42 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
This should be `*` to get CI to run on PRs.

Reply:
You might want to nuke Appveyor at the same time ...

Reply:
The `pull_request` event is there.
There are some events that I don't think should trigger a CI build like `fork`, and that's why the event names are explicit.

Also, we don't want to trigger a CI build for the `gh-pages` branch, so the branch name is also explicit.

I will update the build definition so that it triggers on all branches except `gh-pages`.

Thank you.